Foundation/CGFloat.swift

```diff
@@ -143,6 +143,10 @@ public struct CGFloat {
     
     /// The native value.
     public var native: NativeType
+
+    public var double: Double {
```

Comment:
Understandable. Having been bitten by a 10x regression when `&&` accidentally lost its `@_transparent` for a few weeks, my worry here is about the cost of an additional non-`@_transparent` function call at each of these conversions. Is that silly?